src/psalter.php

```diff
@@ -373,7 +386,16 @@ function (string $line) : bool {
     array_key_exists('dry-run', $options),
     array_key_exists('safe-types', $options)
 );
-$project_analyzer->setIssuesToFix($keyed_issues);
+
+if ($keyed_issues === ['all' => true]) {
+    $project_analyzer->setAllIssuesToFix();
+} else {
+    try {
+        $project_analyzer->setIssuesToFix($keyed_issues);
+    } catch (\Psalm\Exception\UnsupportedIssueToFixException $e) {
+        die($e->getMessage() . PHP_EOL);
```

Comment:
Please use `fwrite(STDERR, 'Message');`

Reply:
Done. Do you think we should change the remaining uses of `die` in this file?

Reply:
I missed those in #1651. Sure, that'd be great! We should also think about using `exit(1)` instead to let the user know that something went wrong.

Reply:
But I'd do that in a new PR.

Reply:
Right, probably best to wait until this is merged before making the new PR to avoid conflicts. I'm going to be busy later today so feel free to make that PR if you want, otherwise I might do it another day.